src/shared/HtmlReady.js

```diff
@@ -144,7 +144,11 @@ function link(state, child) {
         if (state.mutate) {
             // If this link is not relative, http, https, or steem -- add https.
             if (!/^((#)|(\/(?!\/))|(((steem|https?):)?\/\/))/.test(url)) {
-                child.setAttribute('href', 'https://' + url);
+                if (/(mailto:)/.test(url)) {
```

Comment:
Should be a stricter check, otherwise `mailto` anywhere in the URL may conflict with this logic. Can probably just add it to `(steem|https?)`. Also need to verify sanitizeConfig handles `mailto`.

Reply:
agreed. needs a `^` in that regex. also please add tests to `HtmlReady.test.js` - ideally can you add a failing test in one commit and then the fix for the test in a commit after that?

Reply:
also, is it possible to abuse this somehow? should we validate the contents of mailto links?